config.js

```diff
@@ -228,7 +228,7 @@ export default [
       {
         name: 'accessibility concern',
         color: 'D93F0B',
-        description: 'Any bug, feature request or question about the accessibility of a portion of a product',
+        description: 'Any bug, feature request or question about the accessibility of a portion of a product that isn\'t a WCAG failure',
```

Comment:
I had to remove the "any" because it would otherwise not fit the allowed number of characters. Your rephrasing is better, but I wonder if the same restriction applies and not every character will show?

Reply:
It did. Sorted now.